packages/react-button/src/components/ToggleButton/useToggleButtonClasses.tsx

```diff
@@ -11,31 +11,87 @@ export const useToggleButtonClasses = makeVariantClasses<ToggleButtonState>({
       background: 'var(--button-checked-background)',
       color: 'var(--button-checked-contentColor)',
 
+      '@media (forced-colors: active)': {
+        background: 'var(--button-highContrast-checked-background)',
+        color: 'var(--button-highContrast-checked-contentColor)',
+        '.ms-Button-icon': {
+          color: 'var(--button-highContrast-checked-iconColor)',
+        },
+      },
+
       ':hover': {
-        background: 'var(--button-checkedHovered-background)',
-        color: 'var(--button-checkedHovered-contentColor)',
+        background: 'var(--button-checked-hovered-background)',
+        color: 'var(--button-checked-hovered-contentColor)',
+
+        '@media (forced-colors: active)': {
+          background:
+            'var(--button-highContrast-checked-hovered-background, var(--button-highContrast-checked-background))',
+          color:
+            'var(--button-highContrast-checked--hovered-contentColor, var(--button-highContrast-checked-contentColor))',
+          '.ms-Button-icon': {
+            color:
+              'var(--button-highContrast-checked-hovered-iconColor, var(--button-highContrast-checked-contentColor))',
+          },
+        },
       },
 
       ':active': {
-        background: 'var(--button-checkedPressed-background, var(--button-checkedHovered-background))',
-        color: 'var(--button-checkedPressed-contentColor, var(--button-checkedHovered-contentColor))',
+        background: 'var(--button-checked-pressed-background, var(--button-checked-hovered-background))',
+        color: 'var(--button-checked-pressed-contentColor, var(--button-checked-hovered-contentColor))',
+
+        '@media (forced-colors: active)': {
+          background:
+            'var(--button-highContrast-checked-pressed-background, ' +
+            'var(--button-highContrast-checked-hovered-background, ' +
+            'var(--button-highContrast-checked-background)))',
+          color:
+            'var(--button-highContrast-checked--pressed-contentColor, ' +
+            'var(--button-highContrast-checked--hovered-contentColor, ' +
+            'var(--button-highContrast-checked-contentColor)))',
+          '.ms-Button-icon': {
+            color:
+              'var(--button-highContrast-checked-pressed-iconColor, ' +
+              'var(--button-highContrast-checked-hovered-iconColor, ' +
+              '--button-highContrast-checked-contentColor)))',
+          },
+        },
       },
     },
   },
 
   variants: {
+    root: {
+      highContrast: {
```

Comment:
I would expect you need to update `ButtonVariants` typing, so looks like we have a typing bug somewhere

Reply:
`variants` is typed as `any` last time I checked.